src/data/finset/pointwise.lean

```diff
@@ -88,6 +88,13 @@ def singleton_one_hom : one_hom α (finset α) := ⟨singleton, singleton_one⟩
 rfl
 @[simp, to_additive] lemma singleton_one_hom_apply (a : α) : singleton_one_hom a = {a} := rfl
 
+/-- Lift a `one_hom` to `finset` via `image`. -/
+@[to_additive zero_hom.image "Lift a `zero_hom` to `finset` via `image`", simps]
+def one_hom.image [decidable_eq β] [has_one β] [one_hom_class F α β] (m : F) :
```

Comment:
I would much prefer if the function-like `m` were renamed to `f`.

Reply:
Should I also rename `m` in the existing `image_mul`, and `mul_hom.image`?